duos-api/src/matching/matching.service.ts

```diff
@@ -36,7 +47,7 @@ export class MatchingService {
                 await addMatch(likerId, likedId)
                 return {
                     matched: true,
-                    matchedId: likedId
+                    matchedId: likedUser
```

Comment:
not sure if this is relevant but you can just put likedUser.uid here since I don't think the frontend will be doing anything with the matches besides showing a pop-up @dannykhuu2 correct me if I'm wrong here

Reply:
Yea all im going to do is show a pop-up saying matched.

Reply:
I am provided extra info if needed

Reply:
It can be changed to whatever @dannykhuu2 wants

Reply:
it's fine you can just keep it as is then, not that it'll really matter

Reply:
I think it can just be left as likeId, or whatever david suggested